app/pages/clinicadmin/clinicadmin.js

```diff
@@ -577,60 +630,178 @@ export const ClinicAdmin = (props) => {
           <Flex
             px={4}
             py={2}
-            sx={{ borderBottom: baseTheme.borders.default }}
+            sx={{ borderBottom: baseTheme.borders.thick }}
             alignItems={'center'}
           >
-            <Title flexGrow={1}>
-              {t('Clinic Members')}
-            </Title>
+            <Text py={2} sx={{ display: 'block', color: 'text.primary', fontSize: [1, 2, '18px'], fontWeight: 'medium' }}>
+              {t('Workspace Settings')}
+            </Text>
           </Flex>
 
-          <Box mx={4}>
-            {/* Flex Group 1: Search Box and Add Patient button */}
+          <Box mx={4} py={4}>
             <Flex
-              alignItems="center"
-              my={4}
-              justifyContent="space-between"
-              width={['100%', null, 'auto']}
-              sx={{ gap: 2 }}
+              mb={5}
+              p={4}
+              variant="containers.well"
+              sx={{ flexWrap: ['wrap', null,  'nowrap'], gap: 3 }}
             >
-              {isClinicAdmin() && (
-                <Button
-                  id="add-patient"
-                  variant="primary"
-                  onClick={handleInviteNewMember}
-                  fontSize={0}
-                  px={[2, 3]}
-                  lineHeight={['inherit', null, 1]}
-                >
-                  {t('Invite New Clinic Team Member')}
-                </Button>
+              <Box id="clinicWorkspaceDetails" sx={{ flexBasis: ['100%', null, clinic?.ui?.display?.workspacePlan ? '67%' : '100%'], color: 'darkGrey' }}>
+                <Flex mb={2} sx={{ justifyContent: 'space-between', alignItems: 'center', gap: 3, flexWrap: 'wrap' }}>
+                  <Flex sx={{ justifyContent: 'flex-start', color: 'grays.4', gap: 2, alignItems: 'center' }}>
+                    <Icon variant="static" theme={baseTheme} label="Ellipsis for skipped pages" iconSrc={ClinicIcon} />
```

Comment:
This `label` looks like a copy/paste artifact - should probably be updated.

Reply:
Thanks.  Fixed